### DISCUSSION TITLE
In the skin editor, be able to pull elements from other skins.

### DISCUSSION THREAD

**Discussion by SuperSane1202** [ppy/osu#23212] [Ideas] [answered]
hard to explain but in stable skins, skin elements have common filenames so in theory, it should be possible to be able to mix skins in the skin editor by just cutting and replacing skin elements in the skin folder just like how you mix skins in stable. im not a dev or anything like that but it's just an idea so dont rip on me too hard if its dumb 😅

**Comment by Theighlin** [comment, marked as answer]
This is already planned
